test/unit/terraform_loadbalancer_test.go

```diff
@@ -0,0 +1,24 @@
+package unit
+
+import (
+	"testing"
+
+	test_helper "github.com/Azure/terraform-module-test-helper"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_FrontendSubnetIdNotExist(t *testing.T) {
+	vars := map[string]interface{}{
```

Comment:
Why do we set `frontend_subnet_id` in `Test_FrontendSubnetIdNotExist` test method?

Reply:
It's testing against subnet id by name does not exist. Will change the test name.